runtime/runtime/src/verifier.rs

```diff
@@ -101,7 +101,10 @@ pub fn validate_transaction(
     ValidatedTransaction::new(config, signed_tx)
 }
 
-pub fn commit_charging_for_tx(
+/// Set new `signer` and `access_key` in `state_update`.
+///
+/// Note that this does not commit state changes to the on-disk storage.
```

Comment:
nit: `TrieUpdate::commit` also does not commit changes to the `Trie` - `TrieUpdate::finalize` does. So we may say:
```suggestion
/// Note that this does not commit state changes to the `TrieUpdate`.
```